config/deploy/production.rb

```diff
@@ -1,9 +1,15 @@
-set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask(:deploy_host, '')
+set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask('deploy_host (leave off stanford.edu)', '')
```

Comment:
perhaps the prompt should be clear that this is the prod box prefix, not the full prod box name?